test/Models/CswCatalogGroupSpec.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+/*global require,describe,it,expect*/
+var CswCatalogGroup = require('../../lib/Models/CswCatalogGroup');
+
+describe('CswCatalogGroup', function() {
+    it('findLevel function test', function() {
```

Comment:
Bit of a nitpick, but Jasmine specs should generally be named so they read like a specification in English.  So if you have:

```
describe('foo', function() {
    it('should return 4', function() {
    });
});
```

you can read that as "foo should return 4".  Or even include the function name "it should return 4".